iron-ajax.html

```diff
@@ -291,7 +291,14 @@
     },
 
     get requestHeaders() {
-      var headers = Object.create(this.headers || {});
+      var headers = {},
+          header;
+
+      if(this.headers instanceof Object) {
+        for(header in this.headers) {
+          headers[header] = this.headers[header].toString();
```

Comment:
Let's let `xhr` do the type coercion as necessary. Just copy the properties over without `toString`.